pallets/ddc-accounts/src/lib.rs

```diff
@@ -0,0 +1,565 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![recursion_limit = "256"]
+
+use codec::{Decode, Encode, HasCompact};
+
+use frame_support::{
+	parameter_types,
+	traits::{Currency, DefensiveSaturating, ExistenceRequirement},
+	BoundedVec, PalletId,
+};
+pub use pallet_ddc_staking::{self as ddc_staking};
+use scale_info::TypeInfo;
+use sp_runtime::{
+	traits::{AccountIdConversion, AtLeast32BitUnsigned, Saturating, Zero},
+	RuntimeDebug, SaturatedConversion,
+};
+use sp_staking::EraIndex;
+use sp_std::prelude::*;
+
+pub use pallet::*;
+
+/// The balance type of this pallet.
+pub type BalanceOf<T> =
+	<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+
+parameter_types! {
+	/// A limit to the number of pending unlocks an account may have in parallel.
+	pub MaxUnlockingChunks: u32 = 32;
+}
+
+/// Just a Balance/BlockNumber tuple to encode when a chunk of funds will be unlocked.
+#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
+pub struct UnlockChunk<Balance: HasCompact> {
```

Comment:
A bit weird name for a struct. What do you think about `ChunksToUnlock`

Reply:
Good suggestion. Although overall ddc-accounts spec should be prepared, this is just a "temporary implementation"

Reply:
fixed